src/components/tooltip/tooltip.js

```diff
@@ -1,38 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Icon, OverlayTrigger, Popover, Tooltip as PFTooltip } from 'patternfly-react';
+import { Icon } from 'patternfly-react';
+import { Popover, Tooltip as PFTooltip } from '@patternfly/react-core';
 import helpers from '../../common/helpers';
 
-const Tooltip = ({ children, tooltip, id, placement, popover, rootClose, trigger, delayShow, ...props }) => {
+const Tooltip = ({ children, id, placement, isPopover, content, rootClose, trigger, delayShow, ...props }) => {
   const setId = id || helpers.generateId();
 
-  const tooltipPopover = popover ? (
-    <Popover id={setId} {...props}>
-      {popover}
-    </Popover>
-  ) : (
-    <PFTooltip id={setId} {...props}>
-      {tooltip || 'example tooltip'}
-    </PFTooltip>
-  );
+  if (isPopover) {
+    return (
+      <Popover id={setId} hasAutoWidth showClose={false} bodyContent={<div>{content}</div>}>
```

Comment:
required:
- restore `position={placement}`, looks like its used in one spot
- remove the `div` tag from `<div>{content}</div>`, I can't tell it's doing anything, unless you see something?
- remove the `span` tag from `<span>{children || <Icon type="pf" name="info" />}</span>`, I can't tell it's doing anything, unless you see something?